packages/react-hooks/src/useShallowMemo.ts

```diff
@@ -0,0 +1,21 @@
+import * as React from 'react';
+import { shallowCompare } from '@uifabric/utilities';
```

Comment:
For bundle size reasons it might be better to import directly from the file where shallowCompare is defined.